app/Models/User.php

```diff
@@ -80,31 +80,31 @@ public function sendPasswordResetNotification($token)
         'password', 'remember_token',
     ];
 
-    public function games(){
+    public function games() {
         return $this->belongsToMany('App\Models\Game');
     }
 
-    public function news(){
+    public function news() {
         return $this->belongsToMany('App\Models\News');
     }
 
-    public function settings(){
+    public function settings() {
         return $this->hasOne('App\Models\UserSetting', 'user_id', 'id');
     }
 
-    public function logo(){
+    public function logo() {
         return $this->hasMany('App\Models\Logo', 'user_id', 'id');
     }
 
-    public function logovote(){
+    public function logovote() {
         return $this->belongsToMany('App\Models\LogoVote');
     }
 
-    public function userobyx(){
+    public function userobyx() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->